modules/migrating-to-multi-arch-cli.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * updating/migrating-to-multi-payload.adoc
+
+:_content-type: PROCEDURE
+[id="migrating-to-multi-arch-cli_{context}"]
+= Migrating to a cluster with multi-architecture compute machines using the CLI
+
+.Prerequisites
+
+* You must be logged in as a user with `cluster-admin` privileges on all clusters.
```

Comment:
nit: "on all clusters" sounds strange to me.  No worries if there are multi-cluster consumers of this module and we need the plural, but if so, I would expect later plurals where we currently have singulars like "Your `{product-title}` cluster...".  Searching for precedent:

```console
$ git --no-pager grep -oh 'You must.*cluster-admin.*' | sort | uniq -c | sort -n | tail -n5
      2 You must have `cluster-admin` or `project-admin` privileges.
      5 You must be logged in as a user with `cluster-admin` privileges on all clusters.
      5 You must have access to the cluster as a user with the `cluster-admin` role.
      6 You must be logged in as a user with `cluster-admin` privileges.
      6 You must log in to the cluster with a user that has the `cluster-admin` role.
```

The most common three are all  singular (and even then, seems like we still have room for unification, but obviously out of scope for this pull ;))

Reply:
the "have access" phrasing has the useful property of being a statement about the present (access exists right now) vs. "be logged in" which is about state from past action, and doesn't guard against "but the network between your kubeconfig and the cluster is currently flaking" and similar.  And "must log in to the cluster" is requesting immediate action, instead of asking the reader to assess their current environment.  So I very weakly  prefer the "have access" phrasing, if it turns out that nobody else has opinions and we need a tie-breaker ;).

Reply:
Thank you for gathering those! I agree - I prefer "You must have access to the cluster as a user with the `cluster-admin` role." Implementing :)